JS/Lesson32/goksel/Lesson32/my-app/src/components/List/index.jsx

```diff
@@ -1,17 +1,25 @@
 import './styles.css';
+import { TodoContext } from '../../data';
+import { useContext } from 'react';
+
+export const List = () => {
+
+
+	const todo =  useContext(TodoContext);
```

Comment:
The data is in array format, make sure to use a plural form (todos instead of todo).
```suggestion
	const todos =  useContext(TodoContext);
```